word_guess.rb

```diff
@@ -0,0 +1,143 @@
+
+require 'colorize'
+
+class NewGame
+	# method to start new game with randomly selected word
+	def initialize(word)
+		@word = word
+    	@game_status = true
+    	@letter_array = []
+    	@dashes_array = []
+		@incorrect_guesses = 0
+		@already_guessed = []
+		@icecream = ["     ()".colorize(:magenta), "    (__)".colorize(:yellow), "   (____)".colorize(:green), "  (______)".colorize(:cyan), " (________)".colorize(:light_blue), "(__________)".colorize(:red), " X X X X X".colorize(:yellow), "  X X X X ".colorize(:yellow), "   X X X ".colorize(:yellow), "    X X ".colorize(:yellow), "     X".colorize(:yellow)]
+		user_guess
+	end
+
+	def user_guess
+		while @game_status == true do
+			draw_icecream
+        	draw_guesses
+			puts "\nGuess a letter:"
+			guess = gets.chomp.downcase
+			# make sure user enters letter a-z
+			until guess.match(/[a-zA-Z]/)
+				puts "Please guess a letter from a-z:"
+				guess = gets.chomp.downcase
+			end
+			# if already guess letter, reprompt and don't lose round
+			while @already_guessed.include? guess
```

Comment:
While we _can_ use the `include?` method this way, it is preferred to use the parenthesis around the parameter like: `.include?(guess)`